src/controllers/userController.js

```diff
@@ -203,6 +183,12 @@ class userController {
           error: i18n.__('IncorrectEmailPassword'),
         });
       }
+      if (!registered.isUpdated) {
+        return res.status(403).json({
+          status: 403,
+          error: 'Please update your profile information to continue'
```

Comment:
```suggestion
       error:  setLanguage(preferedLang).__('Please_update_your_profile_information_to _continue')
```
@AbdoulNuru Would you please make translation for that response.

You'll have to use such ```setLanguage``` method when I merge. But at least set the locales

Reply:
@hakoemmy I can't use a method that is not there, what is there works I will refactor once you merge your PR, but now I can not use a method that is not available

Reply:
@AbdoulNuru Sure thing